apps/app/src/app/pages/pages/how-to-use/how-to-use.component.html

```diff
@@ -0,0 +1,177 @@
+    prochaines navigations au sein de l'application.
+  </p>
+
+  <h2>Voir la version</h2>
+
+  <p cds-text>
+    La version de l'application est affichée dans la barre de navigation en bas
+    à gauche. Elle permet de donner des informations sur la version de
+    l'application utilisée.
+  </p>
+
+  <p cds-text>
+    Il est important de
+    <strong> transmettre cette version à l'équipe de développement </strong>
+    lors d'une remonté de bogues afin de faciliter la correction de ces
+    derniers.
+  </p>
+</section>
+
+<section *ngIf="isEnglish()">
+  <h1>Understanding and using the application</h1>
+
+  <p>
+    This page allows you to understand all the functionalities present in the
+    interface in order to use it at best.
+  </p>
+
+  <h2>Display data</h2>
+
+  <p cds-text>
+    Different data, such as sessions, tasks, are displayed in different tables
+    in different tables within the application.
+  </p>
+
+  <p cds-text>
+    These tables allow you to view, sort and filter data. A column is sortable
+    as soon as the icon
+    <clr-icon shape="filter-grid" class="is-solid"></clr-icon> to filter is
+    visible. To sort a column, click on the name of the column. the name of the
+    column. To filter a column, click on the icon
+    <clr-icon shape="filter-grid" class="is-solid"></clr-icon>.
+  </p>
+
+  <h2>Navigate between pages</h2>
+
+  <p cds-text>
+    The navigation within the application is done in different ways.
+  </p>
+
+  <ul>
+    <li>
+      Via the dashboard. The latter lists all the applications within within
+      ArmoniK. It is then possible to navigate within the sessions of an
+      application of an application and then within the tasks of a session.
+    </li>
+    <li>
+      Via the tabs. These tabs allow you to quickly switch from one application
+      to application to another once you've been there.
+    </li>
+  </ul>
+
+  <h2>Find the errors</h2>
+
+  <p cds-text>
+    It may happen that an error occurs while using the application. the
+    application. This can be shown by the appearance of a red banner or by the
+    display of an error page.
+  </p>
+
+  <p cds-text>
+    When this happens, it is possible to find out more about the error via the
+    console and the network tab of the browser inspector.
+    <br />
+    When reporting bugs, it is important to
+    <strong> forward this error </strong> as well as the
+    <strong> version number </strong> of the application.
+  </p>
+
+  <h2>Change the language</h2>
+
+  <p cds-text>
+    The application is available in 2 languages: French and English. To change
+    the change it, you just have to click on the corresponding button in the the
+    header of the page, on the top right.
+  </p>
+
+  <p cds-text>
+    The language currently in use is no longer available for selection and its
+    display is is grayed out. Also, your selection is kept in memory for your
```

Comment:
... selection and displayed in grayed out color

Reply:
thanks